queenbee/schema/qutil.py

```diff
@@ -24,7 +24,7 @@ class BaseModel(PydanticBaseModel):
 
     def yaml(self, exclude_unset=False):
         return yaml.dump(
-            self.dict(exclude_unset=exclude_unset),
+            json.loads(self.json(exclude_unset=False)),
```

Comment:
1. Why are we set exclude_unset to False? I found it helpful to set it to True when I want to generate clean YAML files.
2. If you think it should set to False all the time then we have to remove the argument in line 25.

Reply:
This is a typo on my part. Will fix it.